extensions/amp-form/0.1/amp-form.css

```diff
@@ -18,3 +18,28 @@ form.amp-form-submit-success [submit-success],
 form.amp-form-submit-error [submit-error] {
     display: block;
 }
+
+
+.-amp-validation-bubble {
+    background-color: white;
```

Comment:
4. Just noticed my editor has 4-space indent for css, will fix that to 2 instead.